packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1635,27 +1635,24 @@ module.exports = function(log, config, oauthdb) {
     templateName,
     content
   ) {
-    const parsedLink = url.parse(link);
```

Comment:
Though it passes tests, I'm a little leery of this re-implementation. But, it fixes a bug: If a URL already has query params (e.g. like the product download URL for FPN), this code would silently fail to add UTM params at all. The original code was written [about 3 years ago](https://github.com/mozilla/fxa/commit/b22e1d5a4a89a1d316fd1cb50c3511a455435941), so it's possible that node.js API behavior changed since then. Using this new URL API seems to pass all existing tests and add ?utm params as expected to URLs that already have params.

Reply:
Another question, though (maybe for @johngruen): FxA's link processing in emails adds ?utm params to *all* URLs. Do we actually want that for the product download URL?

For example, considering this URL configured for the server:

https://fpn.firefox.com/vpn/download?source=email&env=dev

FxA will use this URL in the email:

https://fpn.firefox.com/vpn/download?source=email&env=dev&product_id=wibble&uid=uid&utm_medium=email&utm_campaign=fx-new-subscription&utm_content=fx-download-subscription

Reply:
It seems reasonable to me not to have the `source=email` param and instead read from the UTMs. What do you think @rlr?

Reply:
totally WFM.

Reply:
> It seems reasonable to me not to have the source=email param and instead read from the UTMs

Okay... this code doesn't add the `?source=email` - that's an ops server config thing for the download URL. So that won't be changed by this PR,